### PR TITLE
Process/scott pd

### PR DESCRIPTION
**Pull request by ayyubibrahimi**
Please review. 

Disregard the changes to baton_rouge_so_cprr.py as I made them before I saw the issue that you created. 

Thanks!

### CODE REVIEW

File: clean/scott_pd_cprr.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+def clean_rank(df):
+    df.loc[:, "rank_desc"] = df.rank_desc.str.lower().str.strip() \
+        .str.replace("pofc", "officer", regex=False)\
+        .str.replace("lt.", "lieutenant ", regex=False)
+    return df
+
+def assign_agency(df):
+    df.loc[:, "agency"] = "Scott PD"
+    return df
```
Comment (pckhoi) [inline clean/scott_pd_cprr.py:34]: checkout `columns.set_values`. You can do stuffs like this without defining an extra function. E.g.
```python
        ...
        .pipe(set_values, {
            'data_production_year': 2020,
            'agency': 'Brusly PD'
        })
```

File: clean/scott_pd_cprr.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        .str.replace("conduct unbecoming/rudeness", "conduct unbecoming (rudeness)", regex=False)
+    return df
+
+def clean():
+    df = pd.read_csv(data_file_path(
+        "scott_pd/scott_pd_cprr_2020.csv"))
+    df = clean_column_names(df) \
+        .rename(columns={
+        'rule_violation': 'charges'})
+    df.columns = ['notification_date', 'full_name', 'charges', 'appeal', 'disposition/action']
```
Comment (pckhoi) [inline clean/scott_pd_cprr.py:53]: I guess you are trying to remove columns? In newer scripts I have stopped doing this as it obscure what columns are removed. We should try to use `drop(columns=[...])` for that purpose instead.

File: clean/scott_pd_cprr.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        .pipe(gen_uid, ["agency", "first_name", "last_name"])\
+        .pipe(gen_uid, ["agency", "uid", "notification_year", "notification_month", "notification_day"], "complaint_uid")
+    return df
+
+if __name__ == "__main__":
+    df = clean()
+    ensure_data_dir("clean")
+    df.to_csv(
+        data_file_path("clean/cprr_scott_pd_2020.csv"),
+        index=False)
```
Comment (pckhoi) [inline clean/scott_pd_cprr.py:76]: All files should have one empty line at the end. Are you familiar with editorconfig? Perhaps we need to create that file to enforce this.

**Comment by pckhoi** [comment]
I guess this isn't ready for review?

File: clean/scott_pd_cprr.py (hunk 1 above)
Comment (ayyubibrahimi) [reply clean/scott_pd_cprr.py:34]: Thanks.

File: clean/scott_pd_cprr.py (hunk 3 above)
Comment (ayyubibrahimi) [reply clean/scott_pd_cprr.py:76]: No. I will look into it now

**Comment by ayyubibrahimi** [comment]
Fixed errors and created fuse script, edited Make file, and added to fuse/all.py and .wrglsync.yaml.

File: data/.wrglsync.yaml [hunk 4]
```diff
@@ -72,6 +74,8 @@ repos:
     fileName: match/cprr_plaquemines_so_2019.csv
   match-lprr-louisiana-state-csc-1991-2020:
     fileName: match/lprr_louisiana_state_csc_1991_2020.csv
+  match-cprr-scott-pd-2020:
+    fileName: match/cprr_scott_pd_2020.csv
```
Comment (pckhoi) [inline data/.wrglsync.yaml:78]: Going forward perhaps we don't need to keep clean and match script in WRGL. Doesn't look like anybody is interested in how those files change.

Comment (rajivsinclair) [reply data/.wrglsync.yaml:78]: Let's discuss. Why do you think we should not? I think it's helpful to keep it all in one place.

Comment (pckhoi) [reply data/.wrglsync.yaml:78]: Maybe I'm wrong but I just feel like we don't have any eyeballs on those files. But we can keep them if you're in doubt.

**Comment by pckhoi** [comment]
@ayyubibrahimi in case you didn't know all PRs need to have one of these labels to be merged: "bug", "documentation", "enhancement".

**Comment by pckhoi** [comment]
@ayyubibrahimi mark this as "Ready for review" when you think it's ready. Which I guess it is?

File: fuse/scott_pd.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        events.OFFICER_HIRE: {'prefix': 'hire', 'keep': ['uid', 'agency']},
+    }, ['uid'])
+    return builder.to_frame()
+
+
+if __name__ == '__main__':
+    cprr = pd.read_csv(
+        data_file_path("match/cprr_scott_pd_2020.csv"))
+    post = prepare_post()
+    per = rearrange_personnel_columns(post)
```
Comment (pckhoi) [inline fuse/scott_pd.py:45]: If CPRR contains officers not found in POST then you should include them. To fuse personnel data from 2 or more datasets look at `lib.personnel.fuse_personnel`

Comment (ayyubibrahimi) [reply fuse/scott_pd.py]: Fused.

**Review by pckhoi — APPROVED** [review]
LGTM

**Pull request merged**